lib/statsd.js

```diff
@@ -95,8 +105,6 @@ function makeStatsD(options, logger) {
         if (typeof originalClose === 'function') {
             originalClose.apply(statsd);
         }
-        statsd._children.forEach(child => child.close());
-        statsd._children = [];
     })(statsd.close);
```

Comment:
We don't need this anymore at all. The whole redefinition of the 'close' method.

Reply:
Removed.